tests/test_format.py

```diff
@@ -28,6 +28,7 @@ def check_file(
     assert_format(source, expected, mode, fast=False)
 
 
+@pytest.mark.filterwarnings("ignore:invalid escape sequence.*:DeprecationWarning")
```

Comment:
this is caused by this example test case:

https://github.com/psf/black/blob/e026701b9a894ef19ad59eb6e394e5f3f7c18fdd/tests/data/simple_cases/docstring.py#L163

at some point a future python version will cause this test to fail

Reply:
or maybe it will just be a warning forever ;) ? https://github.com/python/cpython/issues/77093